scripts/reporting/batch_test_issues.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+##
+##  See COPYING file distributed along with the ncanda-data-integration package
+##  for the copyright and license terms
+##
+
+import argparse
+import pandas as pd
+from pathlib import Path
+import sibispy
+import sibispy.cli
+from sibispy import sibislogger as slog
+import sys
+from typing import Sequence, List, Dict, Mapping, Union
+import yaml
+
+import github
+
+import batch_script_utils as utils
+
+
+def run_batch(verbose, label):
+    if verbose:
+        print(f"Searching for {label}...")
+    base_command = utils.get_base_command(label)
+    id_type = utils.get_id_type(label)
+
+    ncanda_operations = slog.log.postGithubRepo
+    issues = ncanda_operations.get_issues(state="open")
+    scraped_tuples = []
+    for issue in issues[:20]:
```

Comment:
What does the `[:20]` do in this case? Are you just picking the first 20?

Reply:
It's just there when I'm testing this because there are so many issues in test_operations